server/models/handlers/account.js

```diff
@@ -1,9 +1,24 @@
 /* The goal: Take in the user input from the Account Creation form in 
 the front end UI in order to create a user account in the backend*/
 
-async function createUserAccount(firstName, lastName, type, studentID, email, password, phone) {
-    
-    const user = {
+async function createUserAccount(first_name, last_name, type, student_id, email, password, phone) {
+     // Searches the student table to see if a student account exists for the given data.
+
+     If userType = userType.student {
+     const user = await studentModel.insertStudentAccount(first_name, last_name, type, student_id, email, password, phone);
+     if  (!_.isNil(student)) return student;
+     } else if userType = userType.worker {
+     // Searches the worker table to see if a student account exists for the given data.
+     const worker = await workerModel.insertWorkerAccount(first_name, last_name, type, worker_id, email, password, phone);
+     if  (!_.isNil(worker)) return worker;
+     }
+ 
+     // Returns the user
+     return user;
```

Comment:
Hi Nancy,
The user variable declared on line 8 is no longer in scope in line 17. I would recommend declaring the variable outside of the curly braces, perhaps on line 6 and then initializing that variable value in line 8 and line 12.

Reply:
Hi Amy,
Thank you for suggesting that!
My action: I've declared the variable `user` on line 6, which is returned on line 17. 
You can now see these changes in lines 6-17 of server/models/handlers/account.js

Please confirm if this takes care of your suggestion!

Reply:
Hi Nancy,
I would recommend only declaring the variable once using `var` instead of `const` and then only using the variable name (e.g., `user = `) when updating its value rather than `const user = `.

Reply:
Hi Amy,
Thank you for guiding me for that; I have implemented the changes you suggested in lines 6-17 of server/models/handlers/account.js!